include/klee/Support/FloatEvaluation.h

```diff
@@ -12,9 +12,9 @@
 #ifndef KLEE_UTIL_FLOATS_H
 #define KLEE_UTIL_FLOATS_H
 
-#include "klee/util/Bits.h"     //bits64::truncateToNBits
-#include "IntEvaluation.h" //ints::sext
+#include "Bits.h" //bits64::truncateToNBits
 
+#include "klee/Support/IntEvaluation.h" //ints::sext
```

Comment:
This is part of `klee/Support` so `#include "IntEvaluation.h"` should be sufficient. It's weird to do `#include "Bits.h"` but then do `#include "klee/Support/IntEvaluation.h"`.